### DISCUSSION TITLE
Multilingual GitHub-style alert

### DISCUSSION THREAD

**Discussion by vinser** [gethinode/hinode#1133] [Help] [answered]
It would be nice to add multilingual support to GitHub-style alert introduced in v0.25.5.   
To use alerts multilingual we have to translate `Note`, `Tip`, etc

**Comment by markdumay** [comment, marked as answer]
That should already be available using the i18n data:

```
# Alerts
- id: caution
  translation: Caution
- id: important
  translation: Important
- id: note
  translation: Note
- id: tip
  translation: Tip
- id: warning
  translation: Warning
```

Currently only English and Dutch are defined, you’re welcome to submit additional (German) translations!

**Reply by vinser** [reply]
Great! It works, thanks a lot for
